app/views/partials/account_summary/vat/account_summary.scala.html

```diff
@@ -74,7 +74,7 @@ <h2 style="margin-top: 1.875em !important">@messages("vat.your_balance")</h2>
           @messages("vat.we_will_transfer_this_amount_")
           <a id="vat-repayments-account"
             data-journey-click="link - click:VATaccountSummary:repayments bank account"
-            href="@appConfig.getBusinessAccountUrl("manageAccount")#bank">@messages("vat.repayments_bank_account")</a>
+            href="@appConfig.getPortalUrl("vatChangeRepaymentsAccount")(Some(request.vatDecEnrolment))">@messages("vat.repayments_bank_account")</a>
```

Comment:
change this to use the new way we have been used to handle portal urls on BTA

Reply:
:+1: